lib/blogs_api.ex

```diff
@@ -0,0 +1,5 @@
+defmodule BlogsApi do
+  alias BlogsApi.User
+
+  defdelegate create_user(params), to: User.Create, as: :call
```

Comment:
Dos projetos que tive contato aqui, não foi muito comum expormos as funções públicas nesse arquivo do projeto.
Contudo, é uma prática que eu acho muito bacana e evita o uso explícito dos `contexts` dentro da camada de `web`. Uma vez li um [blog post](https://aaronrenner.io/2019/09/18/application-layering-a-pattern-for-extensible-elixir-application-design.html) sobre `Domain Driven Design` no elixir e a pessoa comentava exatamente sobre isso para separar bem o que era `core` e o que era `boundarie`.

Ver aqui me fez refletir porque não é um hábito na trybe haha

Reply:
Bacana!
Obrigada por compartilhar essa informação. 
E o artigo é muito bom. Me ensinou vários pontos importantes! :)